### ISSUE TITLE
Dead link to external website

### ISSUE THREAD

**Issue by pearcetm** [openseadragon/site-build#283]
I noticed that a link to the `threejs` documentation in the "getting started" section, which apparently used to discuss using the local file system or a local dev server, is broken.

https://github.com/openseadragon/site-build/blob/7e4f07ad8d45edfa8e106390428c1682d480119f/doc-home.md?plain=1#L28C269-L29C1

**Comment by iangilman** [comment]
Good catch! Looks like they still have a version of that information on their Installation page:

https://threejs.org/docs/#manual/en/introduction/Installation

...But it's now buried amongst other stuff so probably not a good thing for us to point to.

MDN has a page on this topic:

https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server

Maybe that's a good replacement?

Of course we could just get rid of the link, but I do think it's worth giving people pointers.

In addition, we could point to https://tapiov.net/live-server/ or https://vitejs.dev/.
